src/public/Interfaces.ts

```diff
@@ -9,14 +9,20 @@
  * @returns Output bindings (optional). If you are returning a result from a Promise (or an async function), this 
  * result will be passed to JSON.stringify unless it is a string, Buffer, ArrayBufferView, or number.
  */
-export type AzureFunction = ((context: Context, ...args: any[]) => Promise<any> | void);
+export type AzureFunction<TReturn = unknown> = ((context: Context, ...args: any[]) => Promise<TReturn> | void);
```

Comment:
@alrod - can we do what @G-Rath suggested [here](https://github.com/Azure/azure-functions-nodejs-worker/pull/372#issue-563688769) now that we're bumping typescript version? 
```
export declare type AzureFunction<TReturn = any, TArgs extends any[] = any[]> = ((context: Context, ...args: TArgs) => Promise<TReturn> | void);
```